### PR TITLE
feat(frontend): Setting up React-Bootstrap

### PR DESCRIPTION
**Pull request by ValantineSuh**
Installed and Configured React-Bootstrap for the frontend of the application

-  #15

### CODE REVIEW

**Review by stephane-segning — APPROVED** [review]

File: expense-frontend/src/App.tsx
Comment (Christiantyemele) [inline expense-frontend/src/App.tsx:1]: I think it is better you import bootstrap in the index or main of the application

[the remark this answers is not in the document]
Comment (ValantineSuh) [reply expense-frontend/src/App.tsx:1]: The requested change has been implemented.
Thank you @Christiantyemele  for the correction

File: expense-frontend/src/index.tsx
Comment (Christiantyemele) [inline expense-frontend/src/index.tsx:1]: Good job 👍

**Review by Christiantyemele — APPROVED** [review]

File: expense-frontend/src/index.tsx
Comment (ValantineSuh) [reply expense-frontend/src/index.tsx:1]: Thank you

**Review by bryandino673 — APPROVED** [review]

**Pull request merged**